website/docs/r/alert_channel_gcp_pub_sub.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_gcp_pub_sub"
+description: |-
+  Create and manage GCP Pub Sub Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_gcp\_pub\_sub
+
+You can configure Lacework to forward events to this Google Cloud Pub/Sub asynchronous messaging service using the Lacework Google Cloud Pub/Sub alert channel.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360047496514-Google-Cloud-Pub-Sub).
+
+## Example Usage
+
+```hcl
+resource "lacework_alert_channel_gcp_pub_sub" "example" {
+  name       = "S3 Data Export"
+  project_id = "my-sample-project-191923"
+  topic_id   = "mytopic"
+  credentials {
+    client_id = "client_id"
+    client_email = "foo@example.iam.gserviceaccount.com"
+    private_key = "priv_key"
+    private_key_id = "p_key_id"
+  }
```

Comment:
Could we please update three things here, the topic and project ids should be something meaningful. Also, let's use the same docs on the other resources for the `credentials` field here 👆🏽